server/contexts/logging/logging.go

```diff
@@ -167,13 +171,22 @@ func (l *LoggingContext) Log(ctx context.Context, logger kitlog.Logger) {
 					errs += separator + err.Error()
 				}
 			}
+			var ewuuid fleet.ErrorUUIDer
+			if errors.As(err, &ewuuid) {
+				if uuid := ewuuid.UUID(); uuid != "" {
+					uuids = append(uuids, uuid)
+				}
+			}
 		}
 		if len(errs) > 0 {
 			keyvals = append(keyvals, "err", errs)
 		}
 		if len(internalErrs) > 0 {
 			keyvals = append(keyvals, "internal", internalErrs)
 		}
+		if len(uuids) > 0 {
+			keyvals = append(keyvals, "uuid", strings.Join(uuids, ","))
```

Comment:
Maybe use `strings.Join(uuids, separator))` to keep things consistent?

Reply:
Mhm... is there a good reason to use ` || `? (It does make sense to use such a thing to concatenate multiple errors that may have commas too, but UUIDs have a fixed format and `,` works well AFAICS.)

Reply:
',' works as well.